Natalia_Niemczyk_LaTeX.tex

```diff
@@ -0,0 +1,70 @@
+\documentclass[]{article}
+\usepackage{qtree}
+\usepackage{phonrule}
+\usepackage{tipa}
+\usepackage{ulem}
+\usepackage{starfont}
+\usepackage{hieroglf}
+\usepackage{amsmath}
+\usepackage{amssymb}
+\title{\LaTeX Homework}
+\author{Natalia Niemczyk}
+\date{May 2019}
+\begin{document}
+\maketitle
+\tableofcontents
+\section{My favorite linguistics and computer science topics}
+\begin{enumerate}
+    \item Sentiment analysis
+    \item Web design    
+    \item Game development
+    \item Sociolinguistics
+    \item Phonology    
+    \item Cognitive linguistics
+\end{enumerate}
+\section{About me}
+\textipa{/maI neIm Iz \|[na\|[talja \ae nd aI stAdi n\ae tSr@l l\ae NgwIdZ pr@UsEsIN In g\|[daN\|[sk ||

```

Comment:
1. Where are stress marks???
2. Why strong forms for and, have, can?
3. What kind of vowel do we have in _study_?